src/modules/bigquery.js

```diff
@@ -159,6 +160,40 @@ export default class BigQuery {
       .then((response) => response.data.rows);
   };
 
+  getAutocompleteForColumn = (
+    currText,
+    columnName,
+    dataset,
+    tableName,
+    token,
+    filterStatement,
+    joinStatement,
+  ) => {
+    const url = `https://bigquery.googleapis.com/bigquery/v2/projects/${dataset.dataProject}/queries`;
+    const filterOrEmpty = _.isEmpty(filterStatement)
+      ? `WHERE ${columnName} LIKE '%${currText}%'`
+      : `WHERE ${filterStatement} AND ${columnName} LIKE '%${currText}%'`;
```

Comment:
I think when `filterStatement` is not empty it already starts with "WHERE". Would this cause a parsing error for BigQuery?